crates/driver/src/tests/cases/internalization.rs

```diff
@@ -29,7 +38,10 @@ async fn valid_internalization() {
 async fn untrusted_internalization() {
     let test = setup()
         .pool(ab_pool())
-        .order(ab_order().internalize())
+        .order(ab_order()
+            .owner(eth::H160::from_str(TRADER_ADDRESS).unwrap())
```

Comment:
Since the testing framework seems to assume the same owner for most orders anyway (otherwise we probably wouldn't have a constant for this) I think making this the default and not having to add `.owner()` everywhere would be nicer.
When it's needed `.owner()` could still be used to overwrite the default owner, though.